cli/src/index.js

```diff
@@ -114,19 +114,34 @@ async function processFiles(files) {
   await fsp.mkdir(program.opts().outputDir, { recursive: true });
 
   let decoded = 0;
-  let decodedFiles = await Promise.all(
-    files.map(async (file) => {
-      const image = imagePool.ingestImage(file);
-      await image.decoded;
-      results.set(image, {
-        file,
-        size: (await image.decoded).size,
-        outputs: [],
-      });
-      progress.setProgress(++decoded, files.length);
-      return image;
-    }),
-  );
+  let decodedFiles;
+  try {
```

Comment:
The mozjpeg error kept causing UnhandledPromiseRejections, since it bubbles up to this top level, which does not have an error handler. I'm thinking we should add a top-level `try { } catch {}` that wraps this `processFiles` function.